pombola/south_africa/migrations/0004_parliamentaryterm.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('south_africa', '0003_attendancefororganisationtoggle'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ParliamentaryTerm',
+            fields=[
+                ('number', models.IntegerField(serialize=False, primary_key=True)),
```

Comment:
I don't think we should mix the DB notion of primary key with the parliamentary term number - it's generally considered bad practise even though this is a trivial example.
Can we leave django to make an `id` field that's primary key, and create `number` as a separate `unique` field?